__tests__/app.test.js

```diff
@@ -314,3 +314,22 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (sorting queries)", () => {
+  test("GET:200 sorts the articles by any valid column for ascending or descending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("created_at", { descending: false });
+      });
+  });
+  test("GET:400 responds with an appropriate status and error message when given a non-existent column name", () => {
+    return request(app)
+      .get("/api/articles?sort_by=banana&order=asc")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid request");
+      });
+  });
```

Comment:
This is a good test, but shouldnt need order by added to the end of it if we are only interesting in checking the sort_by value. 

What if I gave an invalid value for order I.e. /api/articles?order=banana, what would you respond with in this case?